test/update.js

```diff
@@ -0,0 +1,29 @@
+var assert = require('assert');
+
+var R = require('..');
+
+describe('update', function() {
+  it('updates the value at the given index of the supplied array', function() {
+    assert.deepEqual(R.update(2, 4, [0, 1, 2, 3]), [0, 1, 4, 3]);
+  });
+
+  it('offsets negative indexes from the end of the array', function() {
+    assert.deepEqual(R.update(-3, 4, [0, 1, 2, 3]), [0, 4, 2, 3]);
+  });
+
+  it('returns the original array if the supplied index is out of bounds', function() {
```

Comment:
We should use `deepEqual` rather than `strictEqual` for the two assertions below. We needn't guarantee that the argument is returned in these cases.